packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -55,16 +57,20 @@
 		outline-offset: 2px;
 	}
 
-	&::after {
-		content: "\2197";
-		font-weight: 400;
+	svg {
 		opacity: 0;
-		position: absolute;
-		right: 0;
-		transition: opacity 0.1s linear;
+		transition: opacity 0.1s ease-in-out;
 		@include reduce-motion("transition");
 	}
 
+	&:hover {
+		svg {
+			opacity: 1;
+			transition: opacity 0.1s ease-in-out;
+			@include reduce-motion("transition");
```

Comment:
Not sure repeating the transition and reduce-motion mixin here is necessary. I think these two lines should be removed, unless I'm missing something.